platform/ios/app/MBXOrnamentsViewController.m

```diff
@@ -1,6 +1,6 @@
 #import "MBXOrnamentsViewController.h"
 
-@import Mapbox;
+#import <Mapbox/Mapbox.h>
```

Comment:
What's the reason for this change? Left over?

Reply:
This was needed to build the iOS SDK 100% on CMake, I will revert the changes before landing this PR.

Reply:
Thanks for the heads-up!